### PR TITLE
Player go boom

### PR DESCRIPTION
**Pull request by aguibert**

### CODE REVIEW

File: frontend/prebuild/src/app/entity/player.ts [hunk 1]
```diff
@@ -49,6 +50,17 @@ export class Player {
 	this.tooltip.visible(isVisible);
   }
   
+  public setStatus(status: string) {
```
Comment (realModusOperandi) [inline frontend/prebuild/src/app/entity/player.ts]: Do a setter

Comment (aguibert) [reply frontend/prebuild/src/app/entity/player.ts]: how do

Comment (realModusOperandi) [reply frontend/prebuild/src/app/entity/player.ts]: `set status(newStatus: string)` then you can just do `player.status = whatever`

Comment (aguibert) [reply frontend/prebuild/src/app/entity/player.ts]: good to know, however when I try this it breaks the login page (which also uses this object).  Going to skip this comment for now

**Pull request merged**